src/configure-balena-host-envvars.sh

```diff
@@ -66,7 +68,13 @@ for index in $(seq 1 ${#HOST_ENVVARS[*]}); do
     VARVALUE=${!VARNAME}
     if [[ -z "$VARVALUE" ]]; then
         VARVALUE=${HOST_VALUES[$index-1]}
-        SUBDOMAIN="$VARVALUE.$RESIN_DEVICE_UUID.$BALENA_TLD"
```

Comment:
This is a major/breaking change btw

Reply:
That's true, it should be a major, though I don't think there's anything that relies on it atm apart from BoB and I think when this initially went in was the week after `BALENA_DEVICE_UUID` had already been added.

Reply:
Changed to Major.